tests/perf-dashboard/python_clientlibs_download.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+from google.cloud import bigquery
+
+DATASET_NAME = 'python_clientlibs_download_by_week'
+VENEER_TABLE_NAME = 'veneer_client_libs'
+STACKDRIVER_TABLE_NAME = 'stackdriver_client_libs'
+GRPC_TABLE_NAME = 'grpc_lib'
+TABLES = [VENEER_TABLE_NAME, GRPC_TABLE_NAME, STACKDRIVER_TABLE_NAME]
+
+CLIENTLIBS = {
+    VENEER_TABLE_NAME: [
+        'google-cloud-core',
+        'google-cloud-speech',
+        'google-cloud-language',
+        'google-cloud-pubsub',
+        'google-cloud-bigquery',
+        'google-cloud-bigtable',
+        'google-cloud-datastore',
+        'google-cloud-spanner',
+        'google-cloud-storage',
+        'google-cloud-vision',
+        'google-cloud-translate',
+        'google-cloud-dns',
+        'google-cloud-videointelligence',
+    ],
+    STACKDRIVER_TABLE_NAME: [
+        'google-cloud-logging',
+        'google-cloud-monitoring',
+        'google-cloud-error_reporting',
+    ],
+    GRPC_TABLE_NAME: [
+        'grpcio',
+    ],
+}
+
+
+def get_weekly_clientlibs_downloads(client_library, date_str):
+    """Use a SQL query to collect the weekly download data of the client
+    libraries.
+
+    Args:
+        client_library (str): key of the client_library in the CLIENTLIBS dict.
```

Comment:
This is parameter is a little confusing, since "client_library" is actually a table name, not a library or package name.  Can you change it to something more explicit like: table_name (str) + project_names (tuple)?